tests/sentry/issues/auto_source_code_config/test_task.py

```diff
@@ -164,52 +190,13 @@ def setUp(self) -> None:
         self.platform = "javascript"
         self.event = self.create_event(
             [
-                {
```

Comment:
Reducing the vertical height of all `create_event` to help read the tests on one page.